config/config.example.js

```diff
@@ -1,6 +1,7 @@
 var config = {
   botToken: "",
   whitelistChats: [],
+  screenConfig: "./config/screens/dsi_raspberry_7inch.js",  
```

Comment:
Can you please define the `screenConfig` option for HDMI as default or leave it out at all? Otherwise HDMI will not be used for new installations because the install script copies the config.example.

From my opinion it would be good if the options were given as comments and `screenConfig`was not defined

_Suggestion_:
```js
...
// change your screen configuration to one of the available options:
// Default: HDMI 
//screenConfig: "./config/screens/hdmi_default.js",
// raspberry 7inch DSI
// screenConfig: "./config/screens/dsi_raspberry_7inch.js",
...
// 
//screenConfig: "./config/screens/dsi_raspberry_7inch.js",
```